docs/docs/deploying-to-digitalocean-droplet.md

```diff
@@ -53,15 +53,15 @@ Follow these instructions for installs on an Ubuntu droplet.
    sudo n stable
    ```
 
-   `Note`: If you check the version now, you would see the older versions of node and npm from the cache.
-   You can either exit and restart your terminal or refresh the cache by following commands,
+   > **Note:** If you check the version now, you would see the older versions of Node.js and npm from the cache.
+   > You can either exit and restart your terminal or refresh the cache by following commands,
 
    ```bash
    hash node
    hash npm
    ```
 
-5. Install the gatsby-cli now globally. This will be useful ahead in building the Gatsby site for production.
+5. Install the Gatsby-CLI now globally. This will be useful ahead in building the Gatsby site for production.
```

Comment:
```suggestion
5. Install the Gatsby CLI globally. This will be useful ahead in building the Gatsby site for production.
```

Reply:
good catch :)

Reply:
i only looked into `dictionary.txt` for reference this time ...